server/src/test/java/com/objectcomputing/checkins/services/onboarding_profile/OnboardingProfileTestUtil.java

```diff
@@ -0,0 +1,101 @@
+package com.objectcomputing.checkins.services.onboarding_profile;
```

Comment:
I believe we will need to move or replicate the tests as well, and they may need modifications.

Reply:
Test file is currently empty because of flyway removal. Ignore this one for now.